README.md

```diff
@@ -0,0 +1,180 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Greetings Rails project Back-end</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Front-end Link](#front-end)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Greetings Rails project Back-end <a name="about-project"></a>
+
+> Greetings Rails project Back-end is the backend of a fun meet and greet project that uses both Rails backend and React with Webpack frontend. If you want to have endless greetings, here is your friend.
+
+## 🛠 Built With <a name="built-with"></a>
+1. Ruby
+2. Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Provide API Endpoint for a React Frontend App
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FRONT END -->
+
+### FRONT END
+
+> View the [Front-end](https://github.com/leloufadel/hello-react-frontend/) part of the project here.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [Optional ] Good Job for added the link to frontend part but the link redirect to the development branch as shown below with the below screenshot but it should be redirect to the feature branch(setup branch) Kindly Fix the issues.
![image](https://github.com/leloufadel/hello-rails-backend/assets/93598189/81b7ed8a-4e54-4f1f-8665-8d94c7bf1190)